client/post-editor/editor-post-formats/accordion.jsx

```diff
@@ -72,10 +72,23 @@ module.exports = React.createClass( {
 				subtitle={ this.getSubtitle() }
 				icon={ <Gridicon icon="types" /> }
 				className={ classes }>
+				<QueryPostFormats siteId={ this.props.site.ID } />
```

Comment:
Not a blocker, but we'll want to move to eliminate the `site` object as a prop, so it'd be better to pass the `siteId` from `getSelectedSiteId` in as a prop (even if duplicating from `site` for now).